pkg/report/linux_test.go

```diff
@@ -145,21 +146,46 @@ func TestLinuxSymbolizeLine(t *testing.T) {
 			"[<ffffffff82d1b1d9>] baz+0x101/0x200\n",
 			"[<ffffffff82d1b1d9>] baz+0x101/0x200 baz.c:100\n",
 		},
-	}
-	symbols := map[string][]symbolizer.Symbol{
-		"foo": {
-			{Addr: 0x1000000, Size: 0x190},
+		// Frame format with module+offset.
+		{
+			"[   50.419727][ T3822] baz+0x101/0x200 [beep]\n",
+			"[   50.419727][ T3822] baz+0x101/0x200 baz.c:100 [beep]\n",
```

Comment:
Nice! Can we have a couple of test cases for PCs that don't have debug info?
Something like:
```
[ 50.419727][ T3822] baz+0x101/0x200 [invalid_module]
[ 50.419727][ T3822] missing_symbol+0x101/0x200 [beep]
[ 50.419727][ T3822] baz+0x300/0x200 [beep]
```